.travis.yml

```diff
@@ -4,13 +4,21 @@ branches:
     - /^v[0-9.]+$/
     - /^maci-.*$/
 language: python
+cache:
+  pip: true
+  directories:
+    - .tox/
+    - $HOME/.cache/pip-faster/
+    - $HOME/.pre-commit/
+before_cache:
+  # We don't currently use any secure environment variables, but just in case, and also to avoid caches that grow forever, remove log dirs.
+  - rm -rf .tox/*/log .tox/log .tox/*/local/log
+  # For some reason (because we build the manpages toxenv inside docker, maybe?), caching .tox/manpages leads to failing itest_% builds.
+  - rm -rf .tox/manpages
 python:
   - "3.6"
-cache: pip
 env:
-  - TOXENV=py36
-  - TOXENV=coverage
```

Comment:
imo there's little reason to have both py36 and coverage, just run coverage always (unlike old versions of coverage, the overhead is essentially zero)